gulpfile.ts

```diff
@@ -104,6 +107,8 @@ async function buildGrammars(): Promise<void> {
 
     fs.copyFileSync(jsonArmGrammarSourcePath, jsonArmGrammarDestPath);
     console.log(`Copied ${jsonArmGrammarDestPath}`);
+    fs.copyFileSync(armConfigurationSourcePath, armConfigurationDestPath);
```

Comment:
I don't understand how changes to the gulpfile are affecting how syntax highlighting works.

Reply:
From the title: "Lang conf file must be copied to dist"

The config file controls this but must be copied into dist in order to be packaged up and actually get distributed with the extension.  It was working locally but not inside VSIX.